Tests/js/views/services/assets/ez-contenteditviewservice-tests.js

```diff
@@ -227,22 +547,29 @@ YUI.add('ez-contenteditviewservice-tests', function (Y) {
             });
             Y.Mock.expect(this.version, {
                 method: 'set',
-                args: ['fields', this.fields]
+                args: ['fields', Y.Mock.Value.Object]
```

Comment:
hum usually it's not enough to test only Mock.Value.*, is this change really needed ?